### DISCUSSION TITLE
How can I change the colour of the title bar?

### DISCUSSION THREAD

**Discussion by Ivanka6669** [fyne-io/fyne#4628] [Q&A] [answered]
How can I customise the title bar? Is it possible to customize buttons or change its colour?

**Comment by Jacalz** [comment, marked as answer]
Not really, no. The operating system (or window manager on Linux/BSD) draws the title bar for us. We do not provide any way to customise it.

**Reply by Ivanka6669** [reply]
Can you tell me if I can remove the title bar and create a mock up of my title bar in the window? I know I can add an exit button to the app, that's the only thing I need. But is there any way to make the app movable?

**Reply by andydotxyz** [reply]
No, that is not supported either. We prefer to provide a consistent user experience, so window decorations are left up to the operating system.
